### ISSUE TITLE
Cannot use spawn eggs from ice and fire

### ISSUE THREAD

**Issue by Sunconure11** [SpongePowered/SpongeForge#1748]
**I am currently running...**

- SpongeForge version: .11.2-2393-6.1.0-BETA-2592
- Forge version: 1.11.2 - 13.20.1.2454
- Java version: 8
- Operating System:

 - [x] Windows

- Plugins/Mods: 

See: https://minecraft.curseforge.com/projects/compitum-magia

**What is the expected result?**

Mob spawns

**What is the current result?**

https://pastebin.com/MUPC7gZX

https://gist.github.com/Sunconure11/382aee4b5cfaab292127c67082dc4811

**Comment by Sunconure11** [comment]
Other things of notes:

Mobs from Mowzies Mobs despawn. This is not intended, plus one of them, upon removal, drops a rather OP item. They are listed as misc with spongeforge.

I've heard claims of dragons from ice and fire despawning.

**Comment by ryantheleach** [comment]
Does the same exception happen when using monster spawn eggs from vanilla / any other mod? 
My (shallow) suspicion is that if vanilla doesn't cause this exception, then the mod differs in how it spawns entities during methods that sponge expects it to place a block instead.

This appears to be a related class, but I'm not familiar enough with Vanilla to compare.

https://github.com/Alex-the-666/Ice_and_Fire/blob/1.11.2/src/main/java/com/github/alexthe666/iceandfire/item/ItemDragonEgg.java

**Comment by Sunconure11** [comment]
The issue does not happen with vanilla or other spawn eggs.

**Comment by Sunconure11** [comment]
After installing the newest version of SpongeForge, I stumbled upon an error that is relevant:

https://pastebin.com/CJ9s4y5N

Happened when I teleported around randomly.

**Comment by phit** [comment]
Closing this issue, as of today 1.10.2 and 1.11.2 are not longer supported. If a sufficiently, decent, and tested Pull Request is made to fix this bug/issue, it will still be considered, however, it is NOT to be expected that any bug fixes made for 1.12.2 will be back ported to 1.11.2 or 1.10.2.

If you can reproduce this issue with SpongeForge on Minecraft 1.12.2 and this issue was wrongly closed, please [open a new issue](https://github.com/SpongePowered/SpongeForge/issues/new) with updated information.

**Comment by Sunconure11** [comment]
I will let you know if this appears on 1.12.2, since I have since moved there, and so has this mod in question. Right now, I'm waiting for a mod to update with a bug fix before I can bring my server back up, as my server currently crashes with Gravestones Extended, and I'm waiting on the dev to fix it.

**Comment by AlexModGuy** [comment]
The mod was updated to 1.12.2 on version 1.3.1.

**Comment by ryantheleach** [comment]
https://github.com/Alex-the-666/Ice_and_Fire/issues/398 indicates the issue still persists on 1.12.

**Comment by Sunconure11** [comment]
I'm noticing it still persists, at least, with the snow villagers from Ice and Fire. In addition, it extends to the travelling merchant from Primitive Mobs. It seems one cannot trade with either unless sponge is removed, either.

**Comment by gabizou** [comment]
@Sunconure11 give SF 3014 a shot. I know it's been a while.

**Comment by Sunconure11** [comment]
I haven't used Sponge in some time, but I'll see about testing this soon.

One unrelated issue I did note was that dragons from Ice and Fire were invisible unless the hitbox settings or whatever were set to high levels. I'd advise having them, by default, be near the highest possible, due to many mods having large mobs and what not.

**Comment by phit** [comment]
![2018-04-23_01 29 59](https://user-images.githubusercontent.com/2097483/39101145-dfe5270c-4695-11e8-9e66-7c6733e3764b.png)
